frontend/scripts/react-components/chart/axis-label/index.js

```diff
@@ -0,0 +1,3 @@
+import Component from './axis-label-component';
```

Comment:
I understand you made this while following the convention inside the `chart/` directory; makes total sense, however if we move this inside the `react-components/dashboard-element/dashboard-widget` directory we should follow the convention that it's used there. Also regarding file names.

I know it's not great, but I prefer having everything follow the same convention until we get to improve it. The `chart/` is an exception because I copy-pasta from another project, and eventually should leave as an independent module _(wishful thinking  someone_ 🙄  _? )._

Reply:
Yes, you're right, I started making the label part of the chart (as `tick`) but after some back and forth I ended up extracting it. My mistake. Will adapt it to follow the convention of `dashboard-widget`'s components.